.github/workflows/build-timestamped-master.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
```

Comment:
Cant we reuse the connector templates for the workflows? @ThisaruGuruge

Reply:
I just copied them from the `h2.driver`. I will check